pages/death.vue

```diff
@@ -0,0 +1,212 @@
+<template>
```

Comment:
have updated number to 152, more have been added to wikidata since this was written originally, compared with query staright from wikidata and is same number

Reply:
have actually calculated the value so that if any more get added to wikidata it will update

Reply:
That's great. Thanks Ruby